src/app/pages/system/advanced/access/access-card/access-card.component.ts

```diff
@@ -163,7 +163,7 @@ export class AccessCardComponent implements OnInit {
   asDuration(tokenLifetime: number): string {
     const duration = intervalToDuration({ start: 0, end: tokenLifetime * 1000 });
     return formatDuration(duration, {
-      format: ['hours', 'minutes', 'seconds'],
+      format: ['days', 'hours', 'minutes', 'seconds'],
```

Comment:
@razvanphp - actual fix is here, we missed `days`, so the problem was that instead of `1 day 1 second` it omitted day and showed `1 second` for example.